home-manager/gnome.nix

```diff
@@ -41,6 +41,7 @@ in
             just-perfection
             dash-to-dock
             # color-picker # Don't enable by default. It conflicts with clipboard-history
+            xremap
```

Comment:
After this

![image](https://github.com/user-attachments/assets/9e3a4b4d-3c71-473c-b1c8-cdb162d89bda)